_glaredb/python.md

```diff
@@ -28,6 +30,18 @@ pip install glaredb
 - import the `glaredb` module
 - `connect` and then either `execute` eagerly or lazily with `sql`
 
+### Intermediates
+
+Using `.sql` to lazily evaluate let's you store a logical plan in a variable so
+that you can incrementally build up queries.
+
+```python
+intermediate = con.sql("select * from tbl where a > 2;")
+
+# note that we reference the variable 'intermediate' here
+con.sql("select * from intermediate where b > 4;")
```

Comment:
```suggestion
con.sql("select * from intermediate where b > 4;").show()
```

So that if someone uses this example as template, it'll actually do something.